conpot/utils/mac_addr.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2015  Adarsh Dinesh <adarshdinesh@gmail.com>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import logging
+import subprocess
+
+logger = logging.getLogger(__name__)
+
+
+def revert(iface):
```

Comment:
Generally I'd put this below the change_mac method so I have already read the change_mac code before I see you using it in revert (maybe also call it revert_mac?).